Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -169,6 +169,27 @@ public extension UIConfig {
             }
         }
 
+        /// Generates tail images for chat message reactions view. This function going to be called a lot.
+        /// Consider either always returning same images or cache function results.
+        ///   - outlineColor: color around tail, it will be main background color most of the time to create
+        ///   effect of cut through message bubble
+        ///   - borderColor: color of tail border
+        ///   - innerColor: filling color of tail bubbles
+        ///   - flipped: `true` for incoming messages, `false` for outgoing messages
+        ///
+        /// Returns: 2 images, `bottom` image will lay under reaction view in hierarchy,
+        /// `top` will lay on top of reaction view in hierarchy
+        public var reactionViewTail: (
+            _ outlineColor: UIColor,
+            _ borderColor: UIColor,
+            _ innerColor: UIColor,
+            _ flipped: Bool
+        ) -> (bottom: UIImage, top: UIImage) = DaVinci.reactionTail(outlineColor:borderColor:innerColor:flipped:)
+
+        /// Generates tail image for big version of chat message reactions view.
+        ///   - flipped: `true` for incoming messages, `false` for outgoing messages
+        public var reactionViewHugeTail: (_ fillColor: UIColor, _ flipped: Bool) -> UIImage = DaVinci.reactionTail(color:flipped:)
+
```

Comment:
Or it can be done by just providing different image inside this closure (same pattern as with tintcolors for highlights), which is way easier than creating/overriding tail class.

Up to you.